pom.xml

```diff
@@ -13,8 +13,7 @@
     <groupId>org.mule.modules</groupId>
     <artifactId>mule-validation-module</artifactId>
     <packaging>mule-extension</packaging>
-    <version>3.0.0-SNAPSHOT</version>
-
+    <version>2.0.3-SNAPSHOT</version>
```

Comment:
Why are you changing the version to 2.0.3?
The version in main, have to point to the next major.

Reply:
done